src/docs/background/payments-overview/index.md

```diff
@@ -0,0 +1,397 @@
+confirming the PaymentIntent are triggered at the same customer action
+for the default process.
+
+**Related Stripe actions:**
+
+- [:action/stripe-create-payment-intent](/references/transaction-process-actions/#actionstripe-create-payment-intent)
+- [:action/stripe-confirm-payment-intent](/references/transaction-process-actions/#actionstripe-confirm-payment-intent)
+
+#### 3. Provider acceptance
+
+When a customer has requested to book a listing, a provider has 6 days
+to accept the booking until it expires automatically. This timeline
+ensures that the Stripe preauthorization does not expire before the
+provider has the opportunity to accept or reject the booking. Once the
+provider accepts the booking, the PaymentIntent is captured and the
+transaction sum is transferred from the customer's account to the
+platform account.
+
+In the
+[FTW Product (Sneakertime)](https://github.com/sharetribe/ftw-product)
+template, the default process combines customer checkout and provider
+acceptance to all be triggered at the same customer action.
+
+**Related Stripe actions:**
+
+- [:action/stripe-capture-payment-intent](/references/transaction-process-actions/#actionstripe-capture-payment-intent)
+
+#### 4. Customer refund
+
+If the customer requests a refund for one reason or another, the
+operator can refund the PaymentIntent. The Flex integration with Stripe
+only supports full refunds. The default integration takes into account
+whether or not the PaymentIntent has already been captured from the
+customer's account.
+
+**Related Stripe actions:**
+
+- [:action/stripe-refund-payment](/references/transaction-process-actions/#actionstripe-refund-payment)
+
+#### 5. Provider payout
+
+Once the booking has completed successfully, the provider's payout is
+paid to the bank account that is linked to their Custom Connect account.
+Depending on how the
+[transaction's line items](/background/pricing/#line-items) have been
+defined, the platform can take a commission of the price from either the
+provider, the customer, or both. The platform is also responsible for
+paying all [Stripe fees](https://stripe.com/en-fi/connect/pricing)
+related to the Custom Connect account usage, so the commissions must be
+defined to cover those expenses as well.
+
+It is important to note that the platform can
+[hold funds on their Stripe account](https://stripe.com/docs/connect/account-balances#holding-funds)
+for up to 90 days - for exceptions see the linked Stripe documentation.
+In other words, the payout must be triggered no more than 90 days after
+the PaymentIntent is created. This means that for booking times
+exceeding 90 days, the process needs to be modified.
+
+<extrainfo title="Manual or automatic payout?">
+In Stripe terms, the Flex integration uses manual payouts. This means
+that Stripe does not automatically pay out the funds on the platform
+account e.g. daily or weekly, and instead the platform controls the payout
```

Comment:
s/platform/Flex

The operator's Stripe account is known as "platform account". Therefore, I think here platform can accidentally be confused with Stripe, or what do you think. We could also include here that the transaction process in Flex controls the payout schedule, but that may be too accurate for this paragraph so maybe stating that Flex controls the payout schedule is enough.

Reply:
Wording clarified